lucene/sandbox/src/java/org/apache/lucene/document/LatLonShape.java

```diff
@@ -94,9 +94,18 @@ private LatLonShape() {
     return new Field[] {new LatLonTriangle(fieldName, lat, lon, lat, lon, lat, lon)};
   }
 
+  /** create a query to find all indexed shapes that comply the {@link QueryRelation} with the provided point
+   **/
+  public static Query newPointQuery(String field, QueryRelation queryRelation, double lat, double lon) {
```

Comment:
do we really need a relation or can we just assume INTERSECTS?

Reply:
Not sure, I would keep it like that for two reason:

- Keep it consistent with all the other queries in LatLonShape

- You can build queries like give me all my shapes that do not contain this point. For Within it became a Term query matching all indexed point which encoded value are equal to the encoded value of the query.